sdk/spring/scripts/spring_managed_external_dependencies.txt

```diff
@@ -624,10 +624,10 @@ com.google.cloud:google-cloud-websecurityscanner;0.118.0
 com.google.cloud:google-cloud-websecurityscanner-bom;0.118.0
 com.google.cloud:libraries-bom;8.1.0
 com.google.code.gson:gson;2.8.6
-com.google.guava:guava;29.0-jre
-com.google.guava:guava-bom;29.0-jre
-com.google.guava:guava-gwt;29.0-jre
-com.google.guava:guava-testlib;29.0-jre
+com.google.guava:guava;30.1-jre
+com.google.guava:guava-bom;30.1-jre
+com.google.guava:guava-gwt;30.1-jre
+com.google.guava:guava-testlib;30.1-jre
```

Comment:
Here is the relationship between spring-boot's version and spring-cloud's version:
| spring-boot   | spring-cloud   |
|----------------|-----------------|
|2.3.8.RELEASE |Hoxton.SR9      |
|2.4.0.RELEASE |2020.0.0           |

The guava's version managed by `spring-cloud-dependencies-Hoxton.SR9.pom` is `29.0-jre`.

Refs: 
1. https://repo.maven.apache.org/maven2/org/springframework/cloud/spring-cloud-dependencies/Hoxton.SR9/spring-cloud-dependencies-Hoxton.SR9.pom
2. https://repo.maven.apache.org/maven2/org/springframework/cloud/spring-cloud-gcp-dependencies/1.2.6.RELEASE/spring-cloud-gcp-dependencies-1.2.6.RELEASE.pom
3. https://repo.maven.apache.org/maven2/com/google/cloud/libraries-bom/13.4.0/libraries-bom-13.4.0.pom

Since `spring-cloud-dependencies-2020.0.0.pom` will not manage guava's version. It's better to update guava's version after we upgrade spring-cloud's version to 2020.0.0.

We already have a branch to update spring-cloud's version: https://github.com/Azure/azure-sdk-for-java/tree/feature/core/reactorupgrade

So could you please merge this PR after `feature/core/reactorupgrade` merged to master branch?

Reply:
Sounds good to me. 👍🏼 29.0 has some issues I wanted to address from our external dependencies. Do you have an ETA for when this will be merged?

Reply:
I think `feature/core/reactorupgrade` will be merged before `2021-02-28`. 
(@alzimmermsft , correct me if I'm wrong.)